From: Chow Loong Jin <hyperair@debian.org>
Date: Sun, 4 Apr 2021 02:08:19 +0800
Subject: Use python3 for generating gtkdoc header

---
 m4/geany-gtkdoc-header.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/geany-gtkdoc-header.m4 b/m4/geany-gtkdoc-header.m4
index 2b6d87f..b8828f1 100644
--- a/m4/geany-gtkdoc-header.m4
+++ b/m4/geany-gtkdoc-header.m4
@@ -21,7 +21,7 @@ AC_DEFUN([GEANY_CHECK_GTKDOC_HEADER],
 	      [test "x$geany_enable_gtkdoc_header" != "xno"],
 	[
 		dnl python
-		AM_PATH_PYTHON([2.7], [have_python=yes], [have_python=no])
+		AM_PATH_PYTHON([3], [have_python=yes], [have_python=no])
 		dnl lxml module
 		AS_IF([test "x$have_python" = xyes],
 		      [have_python_and_lxml=yes
